src/components/tier-row/tier-row.tsx

```diff
@@ -16,8 +16,10 @@ export default function TierRow({ title }: TierRowProps) {
 
   function handleOnDrop(e: React.DragEvent<HTMLDivElement>) {
     const tileJSON = e.dataTransfer.getData("tile");
+    
     if (tileJSON) {
       const tile = JSON.parse(tileJSON);
+      // console.log(tile)
```

Comment:
debug log

Reply:
@sameer1612 . Hi! Sorry, I don't understand. Do I need to delete this commented string?